### DISCUSSION TITLE
Installed PostGIS, getting error when opening spatial_ref_sys table

### DISCUSSION THREAD

**Discussion by piotrpalek** [supabase/supabase#1280] [Questions] [answered]
Hey, I'm trying out supabase and wanted to add a "geography" column to my table, so I've enabled PostGIS in the extensions menu, and added a `point geography(POINT) NOT NULL` column to my table. Now I saw a new `spatial_ref_sys` table in my tables, opened it and got this error:

```
Error: relation "public.spatial_ref_sys" does not exist
```

I gotta mention I'm new to all of this, so it may just be me not understanding something.

**Comment by steve-chavez** [comment, marked as answer]
Hey @piotrpalek,

That one seems to be a bug on the Table Editor UI - which is showing an internal table(`extensions.spatial_ref_sys`) as part of the `public` schema.

PostGIS itself should be fine though, you can query `select * from extensions.spatial_ref_sys limit 1;` to see the table has the usual contents(coordinate systems).

**Reply by piotrpalek** [reply]
Ah gotcha, thanks! :) Should I do an issue for this bug or is it already covered somehow?

**Reply by steve-chavez** [reply]
@piotrpalek A bug report would be most appreciated :)
